docs/v0.6.x/lightning-strategy-examples.md

```diff
@@ -32,22 +32,26 @@ get '/' do
   content_type 'text/html'
 
   redis = Redis.new
-  project = "<your-project-name>"
+  project = '<your-project-name>'
   index_key = params[:index_key] || redis.get("#{project}:index:current")
   redis.get("#{project}:index:#{index_key}")
 end
 ```
 
-The nice thing about this is that you can deploy your app to production, test it out by passing an index_key parameter with the revision you want to test and activate when you feel confident that everything is working as expected.
+The nice thing about this is that you can deploy your application to production, test it out by passing an `index_key` parameter with the revision you want to test and activate when you feel confident that everything is working as expected.
+
+Alternatively, you can use [`ember-cli-deploy-rack`](https://github.com/bitaculous/ember-cli-deploy-rack), which bundles the described functionality in the Sinatra application as a gem.
```

Comment:
@mkempe maybe add a subtitle `### Using with Rack` (or similar) for this so people don't miss it?